### ISSUE TITLE
in generated code: [System.Runtime.CompilerServices.ModuleInitializer] not available for netstandard2.1 (need 5+).

### ISSUE THREAD

**Issue by xiaoyuvax** [Cysharp/MemoryPack#61]
in generated code: [System.Runtime.CompilerServices.ModuleInitializer] not available for netstandard2.1 (need 5+).

> Compiler Error CS0234 The type or namespace name 'name' does not exist in the namespace 'namespace' (are you missing an assembly reference?)

in 
```
  [System.Runtime.CompilerServices.ModuleInitializer]
    public static void RegisterFormatter()
```

**Comment by xiaoyuvax** [comment]
Suggested as below and netstandard2.1 must manually register formatter by calling RegisterFormatter():
```
#if NET5_0_OR_GREATER
    [System.Runtime.CompilerServices.ModuleInitializer]
#endif
    public static void RegisterFormatter()
    {
        if (!MemoryPackFormatterProvider.IsRegistered<global::Wima.Lora.Model.IMan>())
        {
            MemoryPackFormatterProvider.Register(new IManFormatter());
        }
    }
```

**Comment by neuecc** [comment]
Oh, thanks!
I've fixed as you say and released at `v1.8.6`.
